euphoria-core/src/main/java/cz/seznam/euphoria/core/client/operator/Distinct.java

```diff
@@ -71,7 +71,7 @@
         UnaryFunction<IN, ELEM> mapper /* optional */) {
```

Comment:
The following comment has nothing to do with this review, but just a suggestion to start discussion, could we replace these "/* optional */" comments with @Nullable annotation? That seems maybe a little more useful.